data/transition-sites/directgov_firekills.yml

```diff
@@ -6,3 +6,6 @@ tna_timestamp: 20121106112314
 host: firekills.direct.gov.uk
 global: =301 https://www.gov.uk/firekills
 css: directgov
+aliases:
+  - firekills.gov.uk
+  - www.firekills.gov.uk
```

Comment:
Could we add DCLG as an extra organisation for this site? It would be helpful for them to see this in their organisation's list of sites because they are responsible for the DNS for these aliases.